docs/development/local-development.md

```diff
@@ -34,9 +38,11 @@ curl -sL https://deb.nodesource.com/setup_10.x | sudo -E bash -
 curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
 echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
 sudo apt-get update
-sudo apt-get install -y git python-minimal build-essential nodejs yarn
+sudo apt-get install -y git python-minimal build-essential nodejs yarn openjdk-11-jdk
```

Comment:
```suggestion
sudo apt-get install -y git python-minimal build-essential nodejs yarn default-jre-headless
```

[`default-jre-headless`](https://packages.ubuntu.com/bionic/gradle) is enough for ubuntu to run gradle.

Reply:
docs: installing default-jre-headless is sufficient on Ubuntu